index.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <title>Document</title>
+        <link rel="stylesheet" href="style.css">
+    </head>
+    <body>
+        <!-- navbar section -->
+        <section>
+            <header class="navbar">
+                <h4>Rabaya Rabu</h4>
+                <nav>
+                    <div id="navbar"> 
+                        <img src="Images/Menu.png" alt="icon">
+                        <img src="Images/Menu.png" alt="icon"> 
+                        <img src="Images/Menu.png" alt="icon">  
+                    </div>
+                </nav>
+            </header>  
+        </section>
+        <!-- headline section -->
+        <section>
+            <div class="headline">
+                    <h1 id="h1">Hey I am Rabu</h1>
+                
+                    <h4 id="h4">I am a Software Developer</h4>
+    
+                    <p id="p">
+                        I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+                        don’t hestiate to contact me.
+                    </p>
+                    <!-- social media section -->
+                    <ul class="image-icons">
+                        <li><img src="Images/GitHub.png" alt=""></li>
+                        <li><img src="Images/Linkedin.png" alt=""></li>
+                        <li><img src="Images/Angelist.png" alt=""></li>
+                        <li><img src="Images/Twitter.png" alt=""></li>
+                        <li><img src="Images/Medium.png" alt=""></li>
```

Comment:
- [ ] You added the list of social links, but there are no `Images/` folder in your project and the `img` tags cannot load the images making the list not visible. Kindly add the images to this repository so you can load the with the `img` tags.
 - [ ] Also this is a list of social links and each link should be wrapped with the `a` tag so the user can be redirected to your pages. Kindly wrap the images with the `a` tag to make a better user experience.
 - Your design:
 - 
<img width="317" alt="image" src="https://user-images.githubusercontent.com/61919617/221187212-7a871fa3-7ec9-4fa6-ad37-99301ccedf78.png">

Figma template:
<img width="231" alt="image" src="https://user-images.githubusercontent.com/61919617/221187283-c911a40d-4158-47b8-8553-798415ba7f3a.png">